src/main/java/com/homework/week15/sql/generatedb/UserCrudRepository.java

```diff
@@ -0,0 +1,27 @@
+package com.homework.week15.sql.generatedb;
+
+import java.sql.SQLException;
+
+public class UserCrudRepository extends CrudRepository<User> {
+    SqlGenerator sqlGenerator;
+
+    public UserCrudRepository(SqlGenerator sqlGenerator) {
+        super();
+        this.sqlGenerator = sqlGenerator;
+    }
+
+    @Override
+    public User save(User user) {
+
+        try {
+            getConnection().createStatement().executeUpdate(sqlGenerator.generateInsert(user));
+
+        } catch (SQLException e) {
+            e.printStackTrace();
```

Comment:
please rethrow a runtime exception. if error occures you handle it and return a user and no one will know that the user isn't saved in db